src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,6 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
It's important for an interface to have a header comment, since it will be used in many classes.